packages/medusa/src/migrations/1679950221063-category-remove-soft-delete.ts

```diff
@@ -0,0 +1,19 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
```

Comment:
Todo: depending on the decision it is a todo or not :)
Since some user might have tested the categories already and maybe deleted some of them, and that we are deleting the deleted_at, should we also remove all tuple where delete at is not null before dropping the column. In order to avoid the previously deleted categories to pop up again?

Reply:
good catch! <3